pyproject.toml

```diff
@@ -192,14 +192,12 @@ doc = [
     "alabaster",
     "bottle",
     "jinja2<3.1.0", # see https://github.com/readthedocs/readthedocs.org/issues/9037
-    "jupyter-client<7.0",
-    "nbconvert==5.6.1",
-    "nose==1.3.7",
+    "jupyter-client",
+    "nbsphinx",
+    "nose~=1.3.7; python_version < '3.10'",
     "pytest>=6.1",
     "pyx>=0.15",
-    "runnotebook",
-    "scipy<=1.7.1", # see https://github.com/yt-project/yt/issues/3966 and https://github.com/scipy/scipy/issues/16602
-    "sphinx==3.5.4",
+    "sphinx",
```

Comment:
Can we set an explicit lower boundary to document what currently works ?

Reply:
Added current versions as lower bounds to all the packages in `[docs]`